### ISSUE TITLE
Image placeholder

### ISSUE THREAD

**Issue by ermau** [bojanrajkovic/GifWin#43]
Since the gif's can sometimes take a moment to load, it might be nice if we had some kind of place holder to show that your search _did_ yield results.

**Comment by bojanrajkovic** [comment]
Yeah this would be nice. Should be trivial to swap out elements. Do we size the media elements correctly already? If not we should do that too.

**Comment by ermau** [comment]
Not really, but now that #38 is done we can, going to look into doing that.
